src/librustdoc/passes/unindent_comments.rs

```diff
@@ -87,7 +87,7 @@ fn unindent_fragments(docs: &mut Vec<DocFragment>) {
     };
 
     for fragment in docs {
-        if fragment.doc.as_str().lines().count() == 0 {
+        if fragment.doc.as_str().lines().next().is_none() {
```

Comment:
Is there any case where this isn’t equivalent to `fragment.doc.as_str() == “”`?

Reply:
I wasn't sure, but I don't think so. Do you want to ask in `#general` or somewhere else and open a followup PR making that change?

Reply:
Yeah this should just check if it's empty, I'd rather make the change here.
```suggestion
        if fragment.doc == kw::Empty {
```